docs/administrator-docs/installing.md

```diff
@@ -20,11 +20,23 @@ The Jellyfin Docker image is available on [Docker Hub](https://hub.docker.com/r/
     `--name jellyfin \`  
     `--volume /mnt/jellyfin_config:/config \`  
     `--volume /mnt/jellyfin_cache:/cache \`  
-    `--volume /path/to/media:/mnt/library \`  
-    `--publish 8096:8096 \`  
+    `--volume /path/to/media:/media:ro \`  
     `--net=host \`  
-    `jellyfin/jellyfin:latest`
-
+    `jellyfin/jellyfin:latest`  
```

Comment:
This was here before, but `:latest` is redundant - it's used when no tag is specified.

Reply:
I'll remove the `latest` tag. Do you see any value in documenting the different tag options here or just leave it simple and let the user reference the Docker Hub page?

Reply:
No need to document tag options here.